### ISSUE TITLE
System variable mapping

### ISSUE THREAD

**Issue by gdoyle1** [patternfly/patternfly-design#791]
This issue is to create a spreadsheet to map out all global variables in PatternFly. The spreadsheet is here: https://docs.google.com/spreadsheets/d/1T5gllFGnwROOI-03JeLYlxpeOcXn8zVFesVTfv8jq7A/edit?usp=sharing. Once these variables have been mapped (since they are all currently nested), this can take the place of the CSS variables section that appears on patternfly.org. It will give the user a direct mapping of where that variable is used (in what components), etc.

Colors page should be updated once this issue has been completed.

**Comment by gdoyle1** [comment]
As documented on Slack:
Here’s where I’ve been tracking my work: https://github.com/patternfly/patternfly-next/issues/2550. I spoke with Zack about the possibility of replacing the current CSS variable table on the site with something like this. He said it was possible to get it to automatically update, but would take some time. I filed an issue in the React repo for that (with his help). React issue: https://github.com/patternfly/patternfly-react/issues/3521. I followed up with Evan to make him aware that my work on that issue has been completed. Will help the devs out when they pick it up.
